### ISSUE TITLE
Slack channel sign up link or process doesn't work

### ISSUE THREAD

**Issue by jbasko** [awslabs/serverless-application-model#718]
On the github home page of SAM (https://github.com/awslabs/serverless-application-model) at the bottom there is link to https://awssamopensource.splashthat.com/ which says "Join the SAM developers channel (#samdev) on Slack to collaborate with fellow community members and the AWS SAM team."

I have filled this form three times in two weeks time and haven't heard back. When I was attending (online) the Serverless tech talk by Chris Munn earlier this week and reported the issue to the Q&A team in the chat they said the sign up works and that I should check my spam folder. Well there is nothing in my spam folder (apart from millions from Africa I'm yet to claim). Now, I happened to open SAM CLI's homepage where the Slack link is different (https://github.com/awslabs/aws-sam-cli link is to https://awsdevelopers.slack.com/join/shared_invite/ ...). So I'm guessing this one will work, but now it's too late because I decided to not use SAM as I couldn't get hold of anyone and found that SAM isn't production ready, but maybe you can keep some other users if you update the Slack channel link.

Thanks.

**Comment by jlhood** [comment]
I'm so sorry you had that experience attempting to join the SAM slack channel. We'd recently updated the link in the SAM CLI GitHub repo, but missed updating this one. I have a PR up to update it so others don't experience the same thing you did.

We have many customers who currently use SAM in a production setting. I recognize you're very frustrated at this point, but if you wish to share specific feature gaps you see with the current SAM product, we're very open to the feedback.

**Comment by jbasko** [comment]
Thank you. I got an invite the next day.

Perhaps I gave up too soon. The first impressions I had of SAM where:
- the CLI forwarding to `aws cloudformation` without help is weird, I don't know what to pass as I had not used `aws cloudformation` directly before (only through ansible).
- I wanted to implement an API method with apikeyrequired=True and I didn't find in the documentation how to do it yet in CloudFormation documentation it was pretty clear. It seems swagger is forced upon and I didn't need it because I am just using a proxy method.
- didn't see immediately anything clear on how to deploy this with CodePipeline. Now a few weeks later after I have gone through connecting all the bits manually with CloudFormation and custom Lambda based stage deployments I see that some kind of abstraction that allows not to worry about all these details could be useful. But I won't use it because I'd rather have full control if I understand the building blocks.

Thanks and good luck!
